scripts/prepare_sector_network.py

```diff
@@ -158,21 +158,22 @@ def get(item, investment_year=None):
         return item
 
 
-def co2_emissions_year(countries, opts, year):
+def co2_emissions_year(countries, input_eurostat, opts, year):
```

Comment:
Could add emissions scope and report year to function arguments to eliminate snakemake dependency